### PR TITLE
[docs] Added list of preflight skips flags

### PR DESCRIPTION
**Pull request by DaNnNiS**
## Description

Added infromation about preflight skips flag and example of using.

## Why do we need it, and what problem does it solve?

Users will know about flags of preflights skip.
## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: doc
type: fix
summary: Added information about preflight skips flags
impact_level: default 
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/installing/README_RU.md [hunk 1]
```diff
@@ -7,64 +7,94 @@ lang: ru
...
-- *(для кластеров на bare metal и в неподдерживаемых облаках)* ОС сервера находится в [списке поддерживаемых ОС](../supported_versions.html) (или совместима с ними) и до сервера есть SSH-доступ по ключу;
-- *(в поддерживаемых облаках)* наличие квот, необходимых для создания ресурсов, и параметров доступа к облаку (набор зависит от конкретной облачной инфраструктуры или облачного провайдера);
-- доступ до container registry с образами Deckhouse (по умолчанию — `registry.deckhouse.io`).
+Перед установкой убедитесь в следующем:
+
+**Для кластеров на bare-metal и в неподдерживаемых облаках**: сервер использует операционную систему из [списка поддерживаемых ОС](../supported_versions.html) или совместимую с ним, а также доступен по SSH через ключ.
+
+**Для поддерживаемых облаков**: имеются необходимые квоты для создания ресурсов и подготовлены параметры доступа к облачной инфраструктуре (зависят от конкретного провайдера).
+
+**Для всех вариантов установки**: настроен доступ к container registry с образами Deckhouse (по умолчанию — `registry.deckhouse.io`).
```
Comment (z9r5) [inline docs/documentation/pages/installing/README_RU.md:37]:
```suggestion
**Для всех вариантов установки**: настроен доступ к container registry с образами Deckhouse (`registry.deckhouse.io` или `registry.deckhouse.ru`).
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/stage'...

**Review by mortelumina98 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: docs/documentation/pages/installing/README.md [hunk 2]
```diff
@@ -276,30 +315,34 @@ fi
 
 ## Installing Deckhouse
 
-> When you install Deckhouse Enterprise Edition from the official `registry.deckhouse.io` container registry, you must first log in with your license key:
->
-> ```shell
-> docker login -u license-token registry.deckhouse.io
-> ```
+{% alert level="info" %}
+When installing the Enterprise Edition of Deckhouse from the official container registry `registry.deckhouse.io`, you must first authenticate using a license key:
```
Comment (z9r5) [inline docs/documentation/pages/installing/README.md:319]:
```suggestion
When you install Deckhouse Enterprise Edition from the official container registry `registry.deckhouse.io`, you must first log in with your license key:
```

File: docs/documentation/pages/installing/README_RU.md [hunk 3]
```diff
@@ -277,32 +319,36 @@ fi
 
 ## Установка Deckhouse
 
-> При установке Deckhouse редакции Enterprise Edition из официального container registry `registry.deckhouse.io` необходимо предварительно авторизоваться с помощью лицензионного ключа:
->
-> ```shell
-> docker login -u license-token registry.deckhouse.io
-> ```
+{% alert level="info" %}
+При установке Deckhouse редакции Enterprise Edition из официального container registry `registry.deckhouse.io`, необходимо предварительно пройти авторизацию с использованием лицензионного ключа:
```
Comment (z9r5) [inline docs/documentation/pages/installing/README_RU.md:323]:
```suggestion
При установке Deckhouse Enterprise Edition из официального container registry `registry.deckhouse.ru`, необходимо предварительно пройти авторизацию с использованием лицензионного ключа:
```

File: docs/documentation/pages/installing/README_RU.md [hunk 4]
```diff
@@ -277,32 +319,36 @@ fi
...
 
-Запуск контейнера инсталлятора из публичного container registry Deckhouse в общем случае выглядит так:
+```shell
+docker login -u license-token registry.deckhouse.io
```
Comment (z9r5) [inline docs/documentation/pages/installing/README_RU.md:326]:
```suggestion
docker login -u license-token registry.deckhouse.ru
```

File: docs/documentation/pages/installing/README_RU.md [hunk 5]
```diff
@@ -277,32 +319,36 @@ fi
...
+```shell
+docker login -u license-token registry.deckhouse.ru
+```
+
+{% endalert %}
+
+Запуск контейнера инсталлятора Deckhouse из публичного container registry выглядит следующим образом:
 
 ```shell
 docker run --pull=always -it [<MOUNT_OPTIONS>] registry.deckhouse.io/deckhouse/<DECKHOUSE_REVISION>/install:<RELEASE_CHANNEL> bash
```
Comment (z9r5) [inline docs/documentation/pages/installing/README_RU.md:334]:
```suggestion
docker run --pull=always -it [<MOUNT_OPTIONS>] registry.deckhouse.ru/deckhouse/<DECKHOUSE_REVISION>/install:<RELEASE_CHANNEL> bash
```

File: docs/documentation/pages/installing/README_RU.md [hunk 6]
```diff
@@ -312,11 +358,14 @@ docker run -it --pull=always \
   -v "$HOME/.ssh/:/tmp/.ssh/" registry.deckhouse.io/deckhouse/ce/install:stable bash
```
Comment (z9r5) [inline docs/documentation/pages/installing/README_RU.md:358]:
```suggestion
  -v "$HOME/.ssh/:/tmp/.ssh/" registry.deckhouse.ru/deckhouse/ce/install:stable bash
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10995](https://github.com/deckhouse/deckhouse/pull/10995) to the branch [release-1.66](https://github.com/deckhouse/deckhouse/tree/release-1.66) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10996](https://github.com/deckhouse/deckhouse/pull/10996) to the branch [release-1.65](https://github.com/deckhouse/deckhouse/tree/release-1.65) successful!

**Pull request merged**